api/controlplane/v1beta1/k0s_types.go

```diff
@@ -28,6 +28,13 @@ func init() {
 	SchemeBuilder.Register(&K0sControlPlane{}, &K0sControlPlaneList{})
 }
 
+type UpdateStrategy string
+
+const (
+	UpdateStrategyRolling  UpdateStrategy = "Rolling"
+	UpdateStrategyRecreate UpdateStrategy = "Recreate"
```

Comment:
IMO neither of these describe very well the autopilot update way. I mean it's more like `InPlace`. And if we only support one strategy, IMO we should only define one strategy as option

Reply:
Changed it to `InPlace` and remove the second one.